src/app/[locale]/(landing)/page.tsx

```diff
@@ -49,7 +49,7 @@ export default async function LandingPage() {
         <div className="md:flex-grow"></div>
         <a
           href={session ? "/dashboard" : "/read"}
-          className="rounded-lg bg-blue-800 text-white font-bold shadow-md px-4 flex items-center justify-center h-8 md:mt-[4px] ms-1"
+          className="rounded-lg bg-blue-800 text-white font-bold shadow-md px-4 py-0.5 xxs:py-0 flex items-center text-center justify-center w-min xxs:w-fit min-h-8 md:mt-[4px] ms-1"
```

Comment:
I think there is a simpler solution here. If we set `flex-shrink-0` on this anchor element, then the "Global Bible Tools" text will be forced to wrap, instead of compressing the button. That gives us more flexibility for smaller screens and zoom levels